### PR TITLE
Refactor BeforeAfterView to remove test code and implement orientation as a styleable attribute.

### PR DESCRIPTION
**Pull request by leewyatt**
The BeforeAfterView component currently contains some code that was only intended for testing purposes and should be removed to clean up the codebase. Additionally, the existing Orientation property can be replaced with a styleable attribute to enhance flexibility and integration with CSS. - #156

### CODE REVIEW

File: gemsfx/src/main/java/com/dlsc/gemsfx/BeforeAfterView.java [hunk 1]
```diff
@@ -14,27 +19,21 @@
...
-    private static final PseudoClass horizontal = PseudoClass.getPseudoClass("horizontal");
-    private static final PseudoClass vertical = PseudoClass.getPseudoClass("vertical");
+    private static final Orientation DEFAULT_ORIENTATION = Orientation.HORIZONTAL;
+    private static final PseudoClass PSEUDO_CLASS_HORIZONTAL = PseudoClass.getPseudoClass("horizontal");
+    private static final PseudoClass PSEUDO_CLASS_VERTICAL = PseudoClass.getPseudoClass("vertical");
 
     public BeforeAfterView() {
         getStyleClass().add("before-after-view");
 
-        Label before = new Label();
```
Comment (dlemmermann) [inline gemsfx/src/main/java/com/dlsc/gemsfx/BeforeAfterView.java:28]: Why did you remove this? This is not test code, this is the content shown by the view when the application hasn't specified anything. I think this makes sense to have so that it is obvious how the control works. Your thoughts?

**Review by dlemmermann — CHANGES_REQUESTED** [review]

File: gemsfx/src/main/java/com/dlsc/gemsfx/BeforeAfterView.java (hunk 1 above)
Comment (leewyatt) [reply gemsfx/src/main/java/com/dlsc/gemsfx/BeforeAfterView.java:28]: I've just updated the Default node. Please review
